mlir/lib/Conversion/ArithToSPIRV/ArithToSPIRV.cpp

```diff
@@ -1337,7 +1337,9 @@ void mlir::arith::populateArithToSPIRVPatterns(
 
 namespace {
 struct ConvertArithToSPIRVPass
-    : public impl::ConvertArithToSPIRVBase<ConvertArithToSPIRVPass> {
+    : public impl::ConvertArithToSPIRVPassBase<ConvertArithToSPIRVPass> {
+  using ConvertArithToSPIRVPassBase::ConvertArithToSPIRVPassBase;
```

Comment:
Couldn't this and the other passes be `using Base::Base;`? @chelini

Reply:
@kuhar yes, you are right. We can use the shorter form.

Reply:
mind taking a look at: https://github.com/llvm/llvm-project/pull/127756? thanks!